test/lightning_client_mock.go

```diff
@@ -119,3 +127,29 @@ func (h *mockLightningClient) AddInvoice(ctx context.Context,
 
 	return hash, payReqString, nil
 }
+
+func (h *mockLightningClient) SignMessage(ctx context.Context, msg []byte) (
+	string, error) {
+
+	return h.lnd.Signature, nil
```

Comment:
Verify that `msg == signatureMsg`? I don't know how much this mock needs to do, this is just a thought.

Reply:
I don't think the mock should return any errors that couldn't also happen with the real call. I think it's fine to leave it as is for the moment.